README.rst

```diff
@@ -8,14 +8,73 @@ Tensorflow PhaseSpace
    :target: https://tfphasespace.readthedocs.io/en/latest/?badge=latest
    :alt: Documentation Status
 
-Tensorflow implementation of the Raubold and Lynch method for n-body events
+Python implementation of the Raubold and Lynch method for n-body events using
+Tensorflow as a backend.
 
+The code is based on the GENBOD function (W515 from CERNLIB), documented in
 
-* Free software: BSD license
-* Documentation: https://tfphasespace.readthedocs.io.
+    F. James, Monte Carlo Phase Space, CERN 68-15 (1968)
 
+and tries to follow it as closely as possible.
 
-Features
---------
+Detailed documentation, including the API, can be found in https://tfphasespace.readthedocs.io.
+
+Why?
+----
+A statement of need
+
+
+Installing
+----------
+
+To install Tensorflow PhaseSpace, run this command in your terminal:
+
+.. code-block:: console
+
+    $ pip install tfphasespace
+
+This is the preferred method to install Tensorflow PhaseSpace, as it will always install the most recent stable release.
+
+You can also install from the git source by cloning the public repository
+
```

Comment:
I suggest to prevent "accidents" and a too complicated installation to change the instruction below to something like: For the newest development version (in case you really need it), you can install the version from git with `pip install git+https://github.com/zfit/tfphasespace`

And if someone really needs to clone, e.g. for development, the people should anyway know how to do that, right?